README.md

```diff
@@ -97,6 +97,9 @@ Replace `-p 3000:3000` with a port of your choice, `REACT_APP_FM_GATEWAY_API` wi
 
 ## Development
 
+The development password for the gateway is `thereisnosecondbest`.
```

Comment:
Should we just make it `pass` as well? At least it's something one could guess if they missed it in the docs.

Reply:
we use thereisnosecondbest for all the gateway stuff on mutinynet too so I'd suggest just keep it, 'pass' is the odd one out we just set that for all the guardians if you run the gateway dev that automatically sets the a 3/4 federation running